k8s/Makefile

```diff
@@ -42,7 +42,7 @@ CRIO_VERSIONS = v1.26 v1.27 v1.28 v1.29
 # empty). This is useful to distinguish between sysbox-deploy-k8s images that have the
 # same Sysbox version but differ in other components (e.g., crio versions, supported
 # k8s releases, etc.).
-SYSBOX_DEPLOY_K8S_IMAGE_PATCH := 0
+SYSBOX_DEPLOY_K8S_IMAGE_PATCH := ".1"
```

Comment:
Maybe we don't need this for sysbox-ce right?

Reply:
Right, I'll flip this back to ".0" once I'm ready with the release.